sdk/identity/identity/test/internal/node/managedIdentityCredential.spec.ts

```diff
@@ -231,6 +197,117 @@ describe("ManagedIdentityCredential", function() {
     }
   });
 
+  it("sends an authorization request correctly in an Azure Arc environment", async () => {
+    // Trigger App Service behavior by setting environment variables
+
+    process.env.IMDS_ENDPOINT = "https://endpoint";
+
+    const mockFs = require("mock-fs");
```

Comment:
silly question: is this called fs-mock or mock-fs?

Reply:
thanks!